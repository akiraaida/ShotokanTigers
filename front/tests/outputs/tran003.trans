10 John Doe             00000 00000.00 S 
02 John Doe             12345 00005.00   
02 Matt Cow             12346 00005.00   
01 John Doe             12345 00000.05   
00                      00000 00000.00   
10 admin                00000 00000.00 A 
02 John Doe             12345 00005.00   
02 Matt Cow             12346 00005.00   
00                      00000 00000.00   
