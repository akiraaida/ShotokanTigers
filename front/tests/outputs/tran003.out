Please enter a command: 
Please enter your session type: 
Please enter a login name: 
Please enter a command: 
Please enter the transferring account's number: 
Please enter the recipient account's number: 
Please enter amount to transfer: 
Amount has been transferred successfully.
Available Balance (12345):
104.95
Available Balance (12346):
115.00
Daily Transfer Limit Remaining (12345):
995.00
Please enter a command: 
Please enter a command: 
Please enter your session type: 
Please enter a command: 
Please enter the account holder's name: 
Please enter the transferring account's number: 
Please enter the recipient account's number: 
Please enter amount to transfer: 
Amount has been transferred successfully.
Available Balance (12345):
99.95
Available Balance (12346):
120.00
Please enter a command: 
